client/lib/themes/theme_constants.dart

```diff
@@ -0,0 +1,73 @@
+import 'package:flutter/material.dart';
+
+Color primary = const Color(0xFFFF8714);
+Color onprimary = const Color(0xFF000000);
+Color secondary = const Color(0xFFF5F5F5);
+Color onsecondary = const Color(0xFFFFFFFF);
+Color background = const Color(0xFFFFFFFF);
+const apppadding = 16.0;
+
+ThemeData lightTheme = ThemeData(
+  brightness: Brightness.light,
+  primaryColor: primary,
+  floatingActionButtonTheme: FloatingActionButtonThemeData(
+    backgroundColor: primary,
+  ),
+  elevatedButtonTheme: ElevatedButtonThemeData(
+    style: ButtonStyle(
+      shape: MaterialStateProperty.all<OutlinedBorder>(
+        RoundedRectangleBorder(
+          borderRadius: BorderRadius.circular(20.0),
+        ),
+      ),
+      backgroundColor: MaterialStateProperty.all<Color>(primary),
+    ),
+  ),
+  inputDecorationTheme: InputDecorationTheme(
+    border: OutlineInputBorder(
+        borderRadius: BorderRadius.circular(15), borderSide: BorderSide.none),
+    filled: true,
+    fillColor: secondary.withOpacity(1.0),
```

Comment:
An opacity of 1.0 doesn't do anything, why do you need it?

Reply:
solved

Reply:
Then please commit, it please 😀 (don't mark the conversation as resolved until)